api/KeyboardEvent.json

```
@@ -715,7 +715,7 @@
             "status": {
               "experimental": false,
               "standard_track": false,
```

suggestion: (again) since this is specified, let's reflect that in the `standard_track` value

```suggestion
              "standard_track": false,
```

---

Don't see any changes here

---

```suggestion
              "standard_track": true,
```